.github/workflows/create-branch-from-linear.yml

```
@@ -0,0 +1,105 @@
+            echo "LABELS=${{ github.event.inputs.labels }}" >> $GITHUB_ENV
+          else
+            SANITIZED_COMMENT=$(echo "$COMMENT_BODY" | tr '\n' ' ')
+            echo "COMMENT=$SANITIZED_COMMENT" >> $GITHUB_ENV
+            echo "LABELS=$(echo '${{ toJSON(github.event.issue.labels) }}' | jq -r '.[].name' | paste -sd ',' -)" >> $GITHUB_ENV
+          fi
+
+      - name: Extract branch name and determine prefix
+        id: extract
+        run: |
+          TICKET=$(echo "$COMMENT" | grep -oE '[A-Za-z]+-[0-9]+' | head -n1 | tr '[:upper:]' '[:lower:]')
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)